Resources/Prototypes/_Mono/Entities/ShipShield/shield.yml

```
@@ -0,0 +1,109 @@
+          LowPower: {visible: true, state: p1}
+          MediumPower: {visible: true, state: p3}
+          HighPower: {visible: true, state: p6}
+      enum.ContainmentFieldGeneratorVisuals.FieldLight:
+        fieldLight:
+          NoLevel: {visible: false}
+          On: {visible: true, state: a1}
+          OneField: {visible: true, state: a2}
+          MultipleFields: {visible: true, state: a3}
+      enum.ContainmentFieldGeneratorVisuals.OnLight:
+        connectedLight:
+          True: { visible: true }
```

YAMLlint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)